### PR TITLE
Use f-strings and some minor cleanups

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: mkosi [hunk 1]
```diff
@@ -193,7 +193,7 @@ def gpt_root_native(arch: str) -> Tuple[uuid.UUID, uuid.UUID]:
     elif arch == 'aarch64':
         return GPTRootTypePair(GPT_ROOT_ARM_64, GPT_ROOT_ARM_64_VERITY)
     else:
```
Comment (poettering) [inline mkosi]: hmm, the python version check at the top of the script should be updated to 3.6 now, right?

**Comment by poettering** [comment]
lgtm but misses the updated python version check

**Comment by keszybz** [comment]
UPdated.

**Comment by fsateler** [comment]
Setup.py is missing (which is probably where distros will look).

**Comment by lucaswerkmeister** [comment]
> Setup.py is missing (which is probably where distros will look).

#285

**Pull request merged**